### PR TITLE
Fix settings survey route not accessible from mode-context

### PR DESCRIPTION
**Pull request by Ifycode**
#### This pull request makes the following changes
- Fixes ushahidi/platform#4299

#### Details
- Settings module is lazy loaded depending on where user clicks first (mode-bar or mode-context link)

#### Screenshots

|Screenshot 2021-07-23 at 6 05 13 AM|Screenshot 2021-07-27 at 7 18 44 PM|
|--|--|

#### Testing checklist
- [x] Settings module is lazy loaded when "settings" link is clicked on from the mode-bar
- [x] Settings module is lazy loaded when "create new survey" link is clicked on from the mode-context
- [x] Settings module is lazy loaded when "survey settings" button/link is clicked on, from the small dropdown in the mode-context
- [x] I certify that I ran my checklist

Ping @ushahidi/platform

### CODE REVIEW

**Comment by Ifycode** [comment]
@Angamanga can you take a look at this pull request when you have time?

**Comment by Ifycode** [comment]
@Angamanga Another place in the mode-context doesn't work. I've fixed it too (screenshots above).

**Review by Angamanga — APPROVED** [review]
Break out lazy-load function, then approved!

**Comment by Ifycode** [comment]
Ok so moving all those routes from settings-list.routes into the settings.routes file works @Angamanga, but the size for settings bundle is reduced from 1.5mb to about 700kb. The size of the ushahidi-legacy-app bundle is back up to 11.2mb.

**Comment by Ifycode** [comment]
I've made the change @Angamanga. So moving all those routes from settings-list.routes into the settings.routes file works, but the size for settings bundle is reduced from 1.5mb to about 700kb. The size of the ushahidi-legacy-app bundle is back up to 11.2mb. That's what I observed.

**Comment by Angamanga** [comment]
@Ifycode Ok lets go with the original plan for now and skip my idea :D Sorry for the confusion...

**Comment by Ifycode** [comment]
I've pushed a new change where I merged the two ideas @Angamanga you can check and test it now.

File: legacy/app/settings/settings.routes.js [hunk 1]
```diff
@@ -10,19 +10,43 @@ angular.module('ushahidi.settings.routes', [])
...
                 controller: require('./settings.controller.js'),
-                template: require('./settings.html')
-            })
-            .state({
-                name: 'settings.list',
-                url: '/settings',
-                template: require('./settings-list.html'),
-                controller: require('./settings-list.controller.js'),
+                template: require('./settings.html'),
                 lazyLoad: function ($transition$) {
```
Comment (Angamanga) [inline legacy/app/settings/settings.routes.js:14]: Nice solution!

**Review by Angamanga — APPROVED** [review]
Approved!

**Pull request merged**